series-tmdb.en.yaml

```diff
@@ -1722,6 +1808,9 @@ entries:
       # Owarimonogatari
       - season: 4
         anilist-id: 21262
+      # Off & Monster Season
+      - season: 5
```

Comment:
Per [TVDB](https://www.thetvdb.com/series/monogatari/seasons/official/6) this should be season 6, no?

Reply:
Ah I was looking at the wrong file